scripts/helpers.sh

```diff
@@ -98,9 +94,15 @@ pane_content_files_restore_from_archive() {
 
 resurrect_dir() {
 	if [ -z "$_RESURRECT_DIR" ]; then
-		local path="$(get_tmux_option "$resurrect_dir_option" "$default_resurrect_dir")"
+		local path
```

Comment:
Fixes shellcheck lint: Declare and assign separately to avoid masking return values (SC2155)